SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs

```diff
@@ -292,6 +294,18 @@ private void SetInstanceCustomProperties(CustomProperties customProperties)
             _channel.EnqueueAsync(customPropertiesLog);
         }
 
+        private void OnUnhandledExceptionOccurred(object sender, UnhandledExceptionOccurredEventArgs args)
+        {
+            lock (AppCenterLock)
+            {
+                if (_channelGroup != null)
+                {
+                    _channelGroup.ShutdownAsync();
```

Comment:
Can you add a debug log that says "Shutting down App Center due to unhandled exception"?

Reply:
"Shutting down **channel group** due to unhandled exception."